lib/matplotlib/dviread.py

```diff
@@ -859,7 +862,8 @@ def __getitem__(self, texname):
             msg = fmt.format(texname.decode('ascii'), self._filename)
             msg = textwrap.fill(msg, break_on_hyphens=False,
                                 break_long_words=False)
-            matplotlib.verbose.report(msg, 'helpful')
+            log = logging.getLogger(__name__)
+            log.info(msg)
```

Comment:
I would not bother with the call to textwrap, and do the formatting here.

Reply:
Sent a gitter, but I have no idea what this is supposed to do and am not comfortable re-doing the formatting.  I think we could have another PR that cleans up the log messages, but am concerned about doing too many formatting changes unless they are very obvious.

Reply:
It's totally fine.  I think this PR is very close to being mergeable (with the rest to be fixed in a piecemeal fashion later).

Reply:
Phew!  I wasn't looking froward to getting the font library messages all sorted...